src/Workspaces/CSharp/Portable/Extensions/AttributeArgumentSyntaxExtensions.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Linq;
+using System.Threading;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.CSharp.Extensions
+{
+    internal static class AttributeArgumentSyntaxExtensions
+    {
+        /// <summary>
+        /// Returns the parameter to which this argument is passed. If <paramref name="allowParams"/>
+        /// is true, the last parameter will be returned if it is params parameter and the index of
+        /// the specified argument is greater than the number of parameters.
+        /// </summary>
+        public static IParameterSymbol DetermineParameter(
```

Comment:
@AlekseyTs @333fred I can't seem to recall: does `GetOperation(AttributeArgumentSyntax)` return an `IArgumentOperation`? If so, we can replace this method with `IArgumentOperation.Parameter`. If not, it would be good to file a tracking issue.

Reply:
It doesn't look like we have any tests in this area. We should give it a try and see if it works.

Reply:
This method is a copy of [ArgumentSyntaxExtensions.DetermineParameter](https://github.com/dotnet/roslyn/blob/master/src/Workspaces/CSharp/Portable/Extensions/ArgumentSyntaxExtensions.cs#L46). If we change the logic here, we should also change it there.

> It doesn't look like we have any tests in this area. We should give it a try and see if it works.

That's a confusing statement. I really don't know what I'm supposed to do here, especially given that this PR got approved to merge already.